.circleci/config.yml

```diff
@@ -21,10 +21,18 @@ jobs:
       - checkout
       - run: sudo pip3 install -U mypy
       - run: mypy -p capstone360
+  lint:
+    docker:
+      - image: circleci/python
+    steps:
+      - checkout
+      - run: sudo pip3 install -U flake8
+      - run: flake8 --max-line-length=110
 
 workflows:
   version: 2
   build_and_test:
     jobs:
       - test
+      - lint
       # - mypy
```

Comment:
You know better than me here

Reply:
Their docs are decent, if you're interested. https://circleci.com/docs/2.0/config-intro/#section=configuration